changelog/unreleased/features/1987-1992--metrics-metatdata.md

```diff
@@ -0,0 +1,7 @@
+Metrics events now optionally contain a metadata field that is a key-value
+mapping of string to string, allowing for finer-grained introspection. For now
+this enables correlation of metrics events and individual queries. A set of new
+metrics for query lookup use this feature to include the query ID. I.e.:
+```
+{"ts": "2021-12-09T09:42:16.797478769","version": "2021.11.18-266-gbb60334346","actor": "meta-index","key": "meta-index.lookup.candidates","value": 42,"query":"DBECE02A-680C-4FB7-A806-69F262B2DB0A"}
```

Comment:
Is it ok to have longer than 80 chars in the changelog, or should we format it multiline like:

```suggestion
{
  "ts": "2021-12-09T09:42:16.797478769",
  "version": "2021.11.18-266-gbb60334346",
  "actor": "meta-index",
  "key": "meta-index.lookup.candidates",
  "value": 42,
  "query": "DBECE02A-680C-4FB7-A806-69F262B2DB0A"
}
```

Reply:
It's not gonna render correctly either way. Every paragraph gets joined into a single line. After building, you can check `build/CHANGELOG.md` to find how the generated changelog file looks.